java/ql/src/semmle/code/java/frameworks/apache/Lang.qll

```diff
@@ -376,3 +376,28 @@ private class ApacheStrSubstitutorModel extends SummaryModelCsv {
       ]
   }
 }
+
+/**
+ * Taint-propagating models for `StrSubstitutor`.
+ */
+private class ApacheObjectUtilsModel extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        // Note all the functions annotated with `taint` flow really should have `value` flow,
+        // but we don't support value-preserving varargs functions at the moment.
+        "org.apache.commons.lang3;ObjectUtils;false;clone;;;Argument;ReturnValue;value",
+        "org.apache.commons.lang3;ObjectUtils;false;cloneIfPossible;;;Argument;ReturnValue;value",
+        "org.apache.commons.lang3;ObjectUtils;false;CONST;;;Argument;ReturnValue;value",
```

Comment:
Should this also match `CONST_BYTE` and `CONST_SHORT`?

Reply:
I believe these won't work due to a type mismatch? Not 100% sure though

Reply:
Do you mean they won't work for dataflow because CodeQL does not support dataflow with different types? Would it at least be good then to model them as taint?

Reply:
They should work just fine. The type system that's used for pruning data flow allows for implicit casts, it effectively does not distinguish between the different numeric primitive types.

Reply:
Added